### ISSUE TITLE
Cannot use {{# each.}}}

### ISSUE THREAD

**Issue by zhangwuqiao** [sunng87/handlebars-rust#669]
{{#each .}}
sn:{{id}},flowID:{{flow_id}}
{{/each}}

json:
[{"id":1,"flow_id":50},{"id":2,"flow_id":51},{"id":3,"flow_id":52},{"id":4,"flow_id":53}]

     {{#each .}}
   |---------^-
 = reason: invalid handlebars syntax: expected identifier, helper_parameter, block_param, or trailing_tilde_to_omit_whitespace
 
 
![图片](https://github.com/user-attachments/assets/c0d5b6ed-dc6a-4dd4-8ddc-b4e8765305c9)

**Comment by sunng87** [comment]
Yes. This is due to a limitation of our parser. You can use `{{#each this}}` as a workaround
